Gemfile

```diff
@@ -111,7 +111,7 @@ gem 'sanitize', '6.0.2'
 
 # SQLite3 DB driver
 gem 'sqlite3'
-gem 'pg'
+# gem 'pg'
```

Comment:
Are we supporting CE with postgres as the DB? If not, then we should just remove this line

Reply:
AFAIK this is for Heroku only. If we want to continue to support Heroku we can make some tweaks so it'll work with our CE Docker image, but that can be done in a different task. For that reason I'll leave this commented for now.